lib/article_json/elements/image.rb

```diff
@@ -1,16 +1,17 @@
 module ArticleJSON
   module Elements
     class Image < Base
-      attr_reader :source_url, :caption, :float
+      attr_reader :source_url, :caption, :float, :href
 
       # @param [String] source_url
       # @param [Array[ArticleJSON::Elements::Text]] caption
       # @param [Symbol] float
-      def initialize(source_url:, caption:, float: nil)
+      def initialize(source_url:, caption:, float: nil, href: nil)
```

Comment:
Could you please also add the YARDoc `@param` annotation for `href`? Thanks :)

Reply:
Yes, I forgot about it

Reply:
Thanks for adding it :) Ideally, we should also add a bit more description to all those params, but that can be another PR that improves the `@param` annotations all around the project.